zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -493,6 +497,19 @@ pub enum Rw {
         memory_address: u64,
         byte: u8,
     },
+    TxLog {
+        rw_counter: usize,
+        is_write: bool,
+        tx_id: usize,
+        log_id: u64,
+        field_tag: TxLogFieldTag,
+        // topic index if field_tag is TxLogFieldTag:Topic, byte index if field_tag is
+        // TxLogFieldTag:Data
```

Comment:
```suggestion
        // topic index if field_tag is TxLogFieldTag:Topic
        // byte index if field_tag is TxLogFieldTag:Data
```

Reply:
updated in [77fe7a5](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/77fe7a57618c1a13a4cfee7a439daceedc2d4cdf) thanks